modules/MicroMegaWattBSP/umwc_comms/protobuf/lo2hi.pb.c

```diff
@@ -1,5 +1,5 @@
 /* Automatically generated nanopb constant definitions */
-/* Generated by nanopb-0.4.6 */
+/* Generated by nanopb-0.4.5 */
```

Comment:
uMWC originally used nanopb 0.4.6, can we make it common with the Yeti one (0.4.5) or upgrade Yeti to 0.4.6